### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by dakingha69**
This updates the README and adds the clientlib guides to this repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,114 @@
...
+  host: "relay0.testnet.trustlines.network",
+  path: "/api/v1",
+  // ...
+});
+
+// Instance using a relay connected to the TLBC
+const tlbc = new TLNetwork({
+  protocol: "https",
+  wsProtocol: "wss"
+  host: "TODO",
```
Comment (berndbohmeier) [inline README.md]: Left over, or do you want to keep it like this?

File: docs/guides.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+  walletType: "ethers"
+});
+
+const newEthersUser = await laika.user.create();
+await laika.user.loadFrom(newEthersUser);
+```
+
+### Create instance of type `identity`
+
+A user of type `identity` is makes use of meta-transactions.
```
Comment (berndbohmeier) [inline docs/guides.md]: typo

File: docs/guides.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+Therefore the addresses of the deployed [identity factory]() and [implementation]() contracts have to be set.
```
Comment (berndbohmeier) [inline docs/guides.md]: Missing links?

File: docs/guides.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+  wsProtocol: "wss"
+  host: "relay0.testnet.trustlines.network",
+  path: "/api/v1",
+  walletType: "identity",
+  identityFactoryAddress: "0x8D2720877Fa796E3C3B91BB91ad6CfcC07Ea249E",
+  identityImplementationAddress: "0x8BEe92893D3ec62e5B3EBBe4e536A60Fd9AFc9D7",
+});
+
+const newIdentityUser = await laika.user.create();
+await laika.user.loadFrom(newUser);
```
Comment (berndbohmeier) [inline docs/guides.md]: typo? `newIdentityUser`?

File: docs/guides.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+const requests = await tlNetwork2.trustline.getRequests(networkAddress)
+const latestRequest = requests[requests.length - 1]
+const initiatorAddress = latestRequest.from
+const creditlineGivenToInitiator = latestRequest.received
+const creditlineReceivedFromInitiator = latestRequest.given
+
+const { rawTx } = await tlNetwork2.trustline.prepareAccept(
+  networkAddress,
+  initiatorAddress,
+  creditlineGivenToInitiator,
```
Comment (berndbohmeier) [inline docs/guides.md]: Does this work? My IDE reports:
Argument type Amount is not assignable to parameter type number | string

File: docs/guides.md (hunk 3 above)
Comment (berndbohmeier) [reply docs/guides.md]: Created an issue

File: docs/guides.md (hunk 5 above)
Comment (dakingha69) [reply docs/guides.md]: Oh yeah lines 182 and 183 have to be changed to 
```js
const creditlineGivenToInitiator = latestRequest.received.value
const creditlineReceivedFromInitiator = latestRequest.given.value
```

See https://github.com/trustlines-protocol/clientlib/pull/302

**Pull request merged**